.github/workflows/pr-gpu.yaml

```diff
@@ -21,6 +21,11 @@ jobs:
             markers: 'not daily and not remote and gpu and (vision or not vision) and (doctest or not doctest)'
             pytest_command: 'coverage run -m pytest'
             composer_package_name: 'mosaicml'
+          - name: 'gpu-3.10-2.0-vision-doctest'
```

Comment:
can you update daily tests to include this please?

Reply:
it doesn't look like we have any gpu tests in the daily? am i missing something?

Reply:
ah, thank you. done